liberty-maven-plugin/pom.xml

```diff
@@ -238,6 +238,7 @@
                             <artifactId>maven-invoker-plugin</artifactId>
                             <configuration>
                                 <pomExcludes>
+                                    <pomExclude>dev-it/pom.xml</pomExclude>
```

Comment:
Is there a way we can only exclude dev-it for Java 17? It would be nice to let it run for the other Java versions for coverage.